tests/integration_tests/test_integration_elasticsearchstore.py

```diff
@@ -219,3 +219,150 @@ def test_list_all_columns_with_fake_experiment_id(init_store):
+                           'params.param0="valeur4"'),
+                          ([], 'params.param0="test3"'),
+                          (["1e5200ae248b476cb0e60286e3f061a4", "4baa8e505cdb49109b6819a497f1a58a",
+                            "d57a45f3763e4827b7c03f03d60dbbe1"], 'params."param0" LIKE \'%va%\''),
+                          ([], 'params."param0" LIKE \'%tes%\''),
+                          (["1e5200ae248b476cb0e60286e3f061a4"], 'tags.tag2="val2"'),
+                          ([], 'tags.tag0="test3"'),
+                          (["d57a45f3763e4827b7c03f03d60dbbe1"], 'tags."tag0" LIKE \'%Va%\''),
+                          ([], 'tags."tag0" LIKE \'%tes%\''),
+                          (["d57a45f3763e4827b7c03f03d60dbbe1"], 'tags."tag0" LIKE \'%Va%\''),
+                          (["d57a45f3763e4827b7c03f03d60dbbe1"], 'metrics.metric0=0'),
+                          (["1e5200ae248b476cb0e60286e3f061a4"], 'metrics.metric0>0'),
+                          (["1e5200ae248b476cb0e60286e3f061a4", "d57a45f3763e4827b7c03f03d60dbbe1"],
+                           'metrics.metric0>=0'),
+                          (["4baa8e505cdb49109b6819a497f1a58a"], 'metrics.metric0<0'),
+                          (["4baa8e505cdb49109b6819a497f1a58a", "d57a45f3763e4827b7c03f03d60dbbe1"],
+                           'metrics.metric0<=0'),
+                          (["1e5200ae248b476cb0e60286e3f061a4", "4baa8e505cdb49109b6819a497f1a58a",
+                            "d57a45f3763e4827b7c03f03d60dbbe1"], '')])
+@pytest.mark.usefixtures('init_store')
+def test__search_runs_simple_filter(expected_runs_ids, test_filter_string, init_store):
+    actual_runs, next_page_token = init_store._search_runs(experiment_ids=["hjb553MBNoOYfhXjp3Tn"],
+                                                           filter_string=test_filter_string,
+                                                           run_view_type=ViewType.ACTIVE_ONLY)
+    for i, run in enumerate(actual_runs):
+        assert run._info.run_id == expected_runs_ids[i]
+
+
+@pytest.mark.parametrize("expected_runs_ids,test_filter_string",
+                         [(["1e5200ae248b476cb0e60286e3f061a4"],
+                           'params.param1="test3" and tags.tag1="val2" and metrics.metric1=4'),
+                          (["4baa8e505cdb49109b6819a497f1a58a"],
+                           'params."param0" LIKE \'%va%\' and tags."tag1" LIKE \'%vale%\' and'
+                           ' metrics.metric1>9 and metrics.metric0<0'),
+                          ([],
+                           'params.param1="test3" and params."param1" LIKE \'%va%\''),
+                          ([], 'metrics.metric0>0 and metrics.metric0=-1'),
+                          (["1e5200ae248b476cb0e60286e3f061a4"], 'tags.tag1="val2" and'
+                           ' tags."tag1" LIKE \'%va%\'')])
+@pytest.mark.usefixtures('init_store')
+def test__search_runs_complexe_filter(expected_runs_ids, test_filter_string, init_store):
```

Comment:
How do you define 'complexe' ? 

Names are not very meaningful

I suggest 2 solutions:
* The lazy one: Only 1 test called `test_search_filters ` with all test cases parametrized. You can also add line comment if needed to provide more context to a use case (for example 'AND clause test')
* Or: Several test functions... but try to find better names such as 'multiple fiterning CLAUSE'